TestSwiftyDropbox/Podfile.lock

```diff
@@ -1,6 +1,6 @@
 PODS:
   - Alamofire (5.4.3)
-  - SwiftyDropbox (7.0.1):
+  - SwiftyDropbox (8.0.1):
```

Comment:
We're just marking the old method deprecated in this change right? That shouldn't be breaking, so it shouldn't be a major change.

Reply:
this is for something else. i will release this later as a minor change